kew.js

```diff
@@ -42,7 +42,7 @@ if (typeof process !== 'undefined') {
  * @param {Object|undefined} scope
  * @param {Array} args
  */
-function nextTickCallback (defer, callback, scope, args) {
+function nextTickCallback(defer, callback, scope, args) {
```

Comment:
Sorry for the ocd, I thought our style-guide only puts a space if it's an anonymous function, but now I can't find anything like that...